src/rt/tokio.rs

```diff
@@ -212,6 +212,16 @@ where
     }
 }
 
+#[cfg(feature = "client-legacy")]
+impl<T> crate::client::legacy::connect::Connection for TokioIo<T>
+where
+    T: crate::client::legacy::connect::Connection,
+{
+    fn connected(&self) -> crate::client::legacy::connect::Connected {
+        self.inner.connected()
+    }
+}
+
```

Comment:
I would think especially implementations regarding legacy would be better to be located in legacy module.

Reply:
OK. Moved it into the lagacy module.